raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,92 @@
+# Running raiden-cli on the raspberry pi
+
+This tutorial assumes raspberry pi to be in the same LAN as your PC with SSH enabled
```

Comment:
- period missing in the end
- assumes *that*
- network instead of LAN

Reply:
Made some more changes in this.

Reply:
done